users/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const cors = require('cors');
+const { config } = require('./src/config');
+const { logErrors, errorHandler, boomErrorHandler } = require('./src/api/middlewares/error.handler');
+
+const connect2mongo = require('./src/db/connection');
+const routerApi = require('./src/api/routes');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ce-box/CE5508-astraoke-api/pullRequest?prid=8478842)
```suggestion
const routerApi = require("./src/api/routes");
```